debug/gdbserver.py

```diff
@@ -612,69 +615,6 @@ def test(self):
         assertEqual(self.gdb.p("status"), self.crc)
         os.unlink(self.download_c.name)
 
-class MprvTest(GdbTest):
```

Comment:
Isn't *PRIV no longer a register?  It's no longer recognized by binutils, and it had been emulated by OpenOCD for a while to write some other CSRs.